src/web/crate_details.rs

```diff
@@ -187,6 +189,7 @@ impl CrateDetails {
             total_items: total_items.map(|v| v as f32),
             total_items_needing_examples: total_items_needing_examples.map(|v| v as f32),
             items_with_examples: items_with_examples.map(|v| v as f32),
+            features: krate.get("features"),
```

Comment:
We should also strip out 'private' features with underscores and optional dependencies. See some of the discussion in https://internals.rust-lang.org/t/docs-rs-crate-features-display/12895/3.

Reply:
Added, hopefully I did understand the requirements.